bin/p2-exec/main.go

```diff
@@ -214,30 +259,48 @@ func loadEnvDir(dir string) error {
 // a cgroup with the name <cgroupname> will be created, using the parameters for
 // <launchablename> found in the platform configuration
 // then, the current PID will be added to that cgroup
-func cgEnter(platconf, launchableName, cgroupName string) error {
-	platconfBuf, err := ioutil.ReadFile(platconf)
+func cgEnter(config, configKey, cgroupName string, params cgroupParams) error {
+	var configPath string
+	if params.Type == PodCGroup {
+		configPath = "CONFIG_PATH"
```

Comment:
`CONFIG_PATH` is where the user-specified config in their pod manifest ends up. I think the pod-level cgroup settings should go somewhere else in the manifest spec shouldn't it? so probably we want to change the schema of `PLATFORM_CONFIG_PATH` to include pod sizes and launchable sizes

Reply:
btw that will probably unfortunately require a new env variable like `PLATFORM_CONFIG_V2` because currently the schema is a map of launchable name to cgroup config. Probably the top level keys should be `pod:` and `launchables:` something like that